server/src/main/java/org/elasticsearch/index/query/PrefixQueryBuilder.java

```diff
@@ -109,9 +109,6 @@ public String value() {
     }
 
     public PrefixQueryBuilder caseInsensitive(boolean caseInsensitive) {
-        if (caseInsensitive == false) {
-            throw new IllegalArgumentException("The case insensitive setting cannot be set to false.");
-        }
```

Comment:
What's the plan here?  To allow clients to set `caseInsensitive` to `false`?

Reply:
Yep. Makes some of the clients easier to write